apps/remix-ide/src/app/files/compiler-metadata.js

```diff
@@ -29,6 +29,28 @@ class CompilerMetadata extends Plugin {
     return joinPath(path, this.innerPath, contractName + '_metadata.json')
   }
 
+  createHardhatArtifacts (compiledContract, provider) {
+    const contract = compiledContract
+    var hhArtifactsFileName = joinPath('artifacts',contract.file, contract.name + '.json')
```

Comment:
Do you think `_JSONFileName` can be used to process that? also I don't see the `innerPath` being used here.